Tests/test_SeqFeature.py

```diff
@@ -84,6 +85,10 @@ def test_start_before_end(self):
         with self.assertRaisesRegex(ValueError, err):
             FeatureLocation(BeforePosition(42), AfterPosition(23), -1)
 
+        # Features with UnknownPositions should pass test
+        FeatureLocation(42, UnknownPosition())
+        FeatureLocation(UnknownPosition(), 42)
+
```

Comment:
Include some zero positions, and also special case where start and end are equal (e.g. location of digest enzyme cutting site between two bases)?